frontend/website/src/pages/LeaderboardPage.re

```diff
@@ -83,7 +156,10 @@ let make = (~lastManualUpdatedDate) => {
   <Page title="Testnet Leaderboard">
     <Wrapped>
       <div className=Styles.page> <Summary lastManualUpdatedDate /> </div>
-      <ToggleButtons currentOption={state.currentOption} onTogglePress />
+      <div className=Styles.filters>
+        <SearchBar />
```

Comment:
We should make this a controlled component similar to the ToggleButtons component that Martin wrote.